docs/src/dev-docs/building-package.md

```
@@ -78,5 +78,10 @@ task docker-images:package
 
 This will create a Docker image named `clp-package:dev`.
 
+The package includes a `docker-compose.yml` file that can be used to deploy CLP using Docker Compose.
+If you want to manually deploy with Docker Compose instead of using the package scripts, see the 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)